pkgs/development/python-modules/azure-containerregistry/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27
+, azure-core
+, msrest
+, msrestazure
+}:
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "azure-containerregistry";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DIZCHZM5aeKtmJrgwAk5J26ltaxNxKUn3rR+FbmuyZc=";
+    extension = "zip";
+  };
+
+  propagatedBuildInputs = [ azure-core msrest msrestazure ];
+
+  pythonImportsCheck = [ "azure.core" "azure.containerregistry" ];
```

Tests seems to use `pytest`.

```suggestion

  checkInputs = [
    pytestCheckHook
  ];

  pythonImportsCheck = [ "azure.core" "azure.containerregistry" ];
```